README.md

```diff
@@ -1,221 +1,139 @@
-# Uploadcare TinyMCE Plugin
+# File Uploader by Uploadcare
+
+<a href="https://uploadcare.com/?utm_source=github&utm_campaign=uploadcare-tinymce">
+  <img align="right" width="64" height="64"
+    src="https://ucarecdn.com/2f4864b7-ed0e-4411-965b-8148623aa680/uploadcare-logo-mark.svg"
+    alt="">
+</a>
+
+This is a plugin for the [TinyMCE][tinymce] WYSIWYG HTML editor
+providing it to work with [Uploadcare Widget][uc-feature-widget].
 
-That's the [Uploadcare][1] plugin for the [TinyMCE][2] WYSIWYG HTML editor.
 The plugin allows your users to upload files and images
 from local devices, social networks, cloud storages, and more.
 All that — without any backend code that is often required for uploads.
```

Comment:
The plugin allows TinyMCE users to upload media from their devices, social media, cloud storage, and more.
All that without any backend code that's usually required to handle uploads.